pkgs/development/python-modules/metaflow/default.nix

```diff
@@ -0,0 +1,63 @@
+{
+  lib,
+  buildPythonPackage,
+  fetchFromGitHub,
+  boto3,
+  requests,
+  pytestCheckHook,
+  numpy,
+  pytest,
+  setuptools,
+  hypothesis,
+  pytest-xdist,
+}:
+
+buildPythonPackage rec {
+  pname = "metaflow";
+  version = "2.13.9";
+  pyproject = true;
+
+  src = fetchFromGitHub {
+    owner = "Netflix";
+    repo = "metaflow";
+    tag = version;
+    sha256 = "sha256-f/cWZl59i1PRqNqLV5LtcZoDKdovdCKoQJNxzXKczgs=";
+  };
+
+  build-systems = [
+    setuptools
+  ];
+
+  dependencies = [
+    boto3
+    requests
+  ];
+
+  nativeCheckInputs = [
+    pytest-xdist
+    pytestCheckHook
+    hypothesis
+    setuptools
+    numpy
+  ];
+
+  # help with these
+  disabledTestPaths = [
+    "test/data/s3/test_s3.py"
+    "test/unit/test_kubernetes.py"
+    "test/core/tests/card_component_refresh_test.py"
+    "test/core/tests/card_extension_test.py"
+    "test/core/tests/card_refresh_test.py"
+  ];
+
+  pythonImportsCheck = [ "metaflow" ];
+
+  meta = {
+    description = "Open Source AI/ML Platform";
+    homepage = "https://metaflow.org/";
+    changelog = "https://github.com/Netflix/metaflow/releases/tag/${src.tag}";
+    license = lib.licenses.asl20;
+    maintainers = with lib.maintainers; [ kr7x ];
+    platforms = lib.platforms.unix;
```

Comment:
Not necessary for Python packages unless you want to be more restrictive than the default.
```suggestion
```